src/slice.rs

```diff
@@ -544,62 +544,62 @@ impl_slicenextdim_larger!((), Slice);
 #[macro_export]
 macro_rules! s(
     // convert a..b;c into @convert(a..b, c), final item
-    (@parse $dim:expr, [$($stack:tt)*] $r:expr;$s:expr) => {
+    (@parse $dim:expr, [$($concatenate:tt)*] $r:expr;$s:expr) => {
```

Comment:
This rename of the variable inside the macro is an incidental mistake, and should not be in the PR

Reply:
Oh I removed this in my rebase-fix of the branch. No other changes in that branch. :)